cmd/test.go

```diff
@@ -237,7 +247,16 @@ func (t *Test) GetCmd() *cobra.Command {
 			if delay <= 5 {
 				fmt.Printf("Warning: delay is set to %d seconds, incase your app takes more time to start use --delay to set custom delay\n", delay)
 				if isDockerCmd {
-					fmt.Println("Example usage:\n", `keploy test -c "docker run -p 8080:808 --network myNetworkName myApplicationImageName" --delay 6\n`)
+					fmt.Println("Example usage:\n", `keploy test -c "docker run -p 8080:8080 --network myNetworkName myApplicationImageName" --delay 6\n`)
+				} else {
+					fmt.Println("Example usage:\n", cmd.Example)
+				}
+			}
+
+			if isDockerCmd && buildDelay <= 30*time.Second {
+				fmt.Printf("Warning: buildDelay is set to %d, incase your docker container takes more time to build use --buildDelay to set custom delay\n", buildDelay)
+				if isDockerCmd {
+					fmt.Println("Example usage:\n", `keploy test -c "docker-compose up --build" --buildDelay 35s\n`, "\nor\n", `keploy test -c "docker-compose up --build" --buildDelay 1m\n`)
 				} else {
 					fmt.Println("Example usage:\n", cmd.Example)
```

Comment:
Since this `buildDelay` flag is only in the case of the docker command, there is no need to add this example here.

Reply:
> Since this `buildDelay` flag is only in the case of the docker command, there is no need to add this example here.

By default, do we assume that the docker environment is built when the user uses the `record` command? Do we consider the situation where the user needs to rebuild when running the `test` command? Then the buildDelay parameter still needs to be checked

Reply:
@IllTamer Whenever there is a docker env the `isDockerCmd` is true since the keploy Dockerfile contains this as an [environment variable](https://github.com/keploy/keploy/blob/main/Dockerfile#L24). So if you've checked that it is dockerCmd & the buildDelay is <=30 then just show the warning as the usage rather than showing another example.